packages/widgets/src/types.ts

```diff
@@ -32,7 +32,12 @@ export type WidgetSettings = {
   /** Widget subtitle */
   subtitle: TextSettings;
   /** Legend component settings */
-  legend?: LegendSettings;
+  legend?: LegendSettings & BubbleWidgetLegendSettings;
```

Comment:
The `&` stands for type union. We need to use alternative here so the legend types should look like:
```
interface LegendSettings {}

interface BubbleWidgetLegendSettings extends LegendSettings { ... }

legend?: LegendSettings | BubbleWidgetLegendSettings;

```